tests/test_estimation.py

```diff
@@ -10,6 +10,9 @@
 
 cdnow_customers = load_cdnow()
 cdnow_customers_with_monetary_value = load_summary_data_with_monetary_value()
+returning_cdnow_customers_with_monetary_value = cdnow_customers_with_monetary_value[
+    cdnow_customers_with_monetary_value['frequency'] > 0
+]
```

Comment:
I moved it into the test. The zero values come from Hardie's spreadsheet -- it's because they calculate the mean value of *repeat* transactions. Not sure why the model can't just use the mean transaction value. The answer may be in the original paper, which I haven't read yet (can't find it in http://www.brucehardie.com/notes/025/gamma_gamma.pdf)